test/e2e/scheduledjob.go

```diff
@@ -46,6 +47,19 @@ var _ = framework.KubeDescribe("ScheduledJob", func() {
 	f := framework.NewFramework("scheduledjob", options, nil)
 
 	BeforeEach(func() {
+		// ScheduledJob was renamed to CronJob for the 1.5
+		// release. However, the ScheduledJob endpoint still
+		// exists, but returns CronJob objects. These tests
+		// can't work with that.
+		v := version.MustParse("1.5.0")
+		gte, err := framework.ServerVersionGTE(v, f.Client)
+		if err != nil {
+			framework.Failf("Failed to get server version: %v", err)
+		}
+		if !gte {
+			framework.Skipf("Not supported for server versions after %q", v)
+		}
+
```

Comment:
But that's exactly what it does, it checks if the server version is at least X. Though I agree, the naming is unfortunate. Compare [these lines](https://github.com/kubernetes/kubernetes/blob/29d10a817ae6eadfd271447a2ab6bbe50856d458/test/e2e/framework/util.go#L410-L418) with your code.